build/VStudio/installer/Product.wxs

```diff
@@ -125,10 +125,10 @@
 
             <!-- install assembly -->
             <Component Id="C.$(var.MyProductFileName)_msil.dll" Guid="0D8BA6AE-9F87-402B-AE1A-95B0AE3BE179">
-                <File Id="FILE.$(var.MyProductFileName)_msil.dll" Name="$(var.MyProductFileName)-msil.dll" KeyPath="yes" />
+                <File Id="FILE.$(var.MyProductFileName)_msil.dll" Source="..\build\$(var.Configuration)\$(var.MyProductFileName).net\net35\" Name="$(var.MyProductFileName)-msil.dll" KeyPath="yes" />

```

Comment:
Is the `winfsp-msil.dll` file from `net35` usable by all variants of .NET? If not then the installer needs to change to include DLLs for the additional .NET variants that we wish to support.